ng-dev/misc/BUILD.bazel

```diff
@@ -11,5 +11,9 @@ ts_library(
         "@npm//@types/node",
         "@npm//@types/yargs",
         "@npm//chalk",
+        "@npm//license-checker",
+        "@npm//@types/license-checker",
+        "@npm//spdx-satisfies",
+        "@npm//@types/spdx-satisfies",
```

Comment:
Can we have a fine-grained target for the new sub-command? We should start doing that incrementally now.